src/library/dlgtrackinfo.cpp

```diff
@@ -484,6 +500,16 @@ void DlgTrackInfo::loadTrack(const QModelIndex& index) {
     }
 }
 
+void DlgTrackInfo::focusField(const QString& property) {
+    if (property.isNull() || property.isEmpty()) {
```

Comment:
isEmpty() includes isNull()
```suggestion
    if (property.isEmpty()) {
```